src/consensus.rs

```diff
@@ -151,19 +149,12 @@ impl<T: Proposition> Consensus<T> {
                 "[{}] Detected super majority over super majorities: {proposals:?}",
                 self.id()
             );
-            let votes = crate::vote::simplify_votes(&self.votes.values().cloned().collect());
             let decision = Decision {
-                votes,
+                generation: signed_vote.vote.gen,
                 proposals,
-                faults: self.faults(),
             };
-            let vote = self.build_super_majority_vote(
-                decision.votes.clone(),
-                decision.faults.clone(),
-                signed_vote.vote.gen,
-            )?;
             self.decision = Some(decision);
-            return Ok(VoteResponse::Broadcast(vote));
+            return Ok(VoteResponse::WaitingForMoreVotes);
```

Comment:
Why now don't need to broadcast anymore?

Reply:
This turned out to be a bug! reverted